### PR TITLE
Extra information

### PR DESCRIPTION
**Pull request by abUwUser**
This fork adds:

### Mobile/Android
- [**ReVanced**](https://github.com/revanced/revanced-manager) - Android app patcher that provides patches to popular apps such as youtube.
- [**Aliucord**](https://github.com/Aliucord/Aliucord) - Plugin-based Discord mod for Android.
### Mobile/iOS <sup>`NEW!`</sup>
- [**Enmity**](https://github.com/revanced/revanced-manager) - Plugin-based Discord mod for iOS, similar to Aliucord.
### Utilities/Misc
- [**KDE Connect**](https://kdeconnect.kde.org/) - Powerful application to integrate your phone with your computer and vice-versa.
- [**OBS (Open Broadcaster Software)**](https://obsproject.com/) - Open source application for streaming and recording.
- [**Kdenlive**](https://kdenlive.org/en/) - Open source free video editor.
### PC-Software/Software
- [**Microsoft Activation Scripts**](https://github.com/massgravel/Microsoft-Activation-Scripts) - Open source activator that includes various scripts for Windows and Office.

### CODE REVIEW

**Comment by kekkodance** [comment]
you can remove the ios section, it's pointless to add it now as we have no maintainers or info for it, also in ReVanced you can make youtube uppercase

**Comment by kekkodance** [comment]
the description for MAS is also a bit vague

**Comment by Helmasko** [comment]
We were meant to be making a Windows section that was meant to be a replacement for /Utilities/Debloating/ and have all Windows related things there including MAS, but thats still not done and probably won't be in a long while. I think we should still add MAS to /Utilities/Debloating/ rather than /Software, also replace MAS link with the official website

**Comment by abUwUser** [comment]
> you can remove the ios section, it's pointless to add it now as we have no maintainers or info for it, also in ReVanced you can make youtube uppercase

the iOS section is more of a motivational thing to people add new stuff to there, but if you believe it is unnecessary then its okay

**Comment by abUwUser** [comment]
> We were meant to be making a Windows section that was meant to be a replacement for /Utilities/Debloating/ and have all Windows related things there including MAS, but thats still not done and probably won't be in a long while. I think we should still add MAS to /Utilities/Debloating/ rather than /Software, also replace MAS link with the official website

well, it fits better to be at /Software than /Utilities/Debloating because that section is dedicated to debloating Windows, not activating. later on, when the Windows section gets added, it could be moved to there

**Comment by abUwUser** [comment]
hello?

**Comment by abUwUser** [comment]
is there anything that you guys may suggest removing?

**Comment by kekkodance** [comment]
all good! @Helmasko all good?

File: Mobile/iOS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: iOS 
+description: List of useful iOS mobile apps.
+---
+
+## Mods
+[**Enmity**](https://github.com/revanced/revanced-manager) - Plugin-based Discord mod for iOS, simmilar to Aliucord.
```
Comment (Helmasko) [inline Mobile/iOS.md:7]: tf is this link

Comment (kekkodance) [reply Mobile/iOS.md:7]: fix ig

Comment (abUwUser) [reply Mobile/iOS.md]: fixed on 79f4c8b

**Comment by abUwUser** [comment]
Anything more?

**Comment by Helmasko** [comment]
Nope, thanks for the contribution.

**Pull request merged**